### ISSUE TITLE
Feature ：I hope this tool can ....... 

### ISSUE THREAD

**Issue by 520f** [scaljeri/oh-my-mock#144]
- Adding, replacing and removing request headers.
- Replacing the request body, text/JSON, Base64 or form data.

**Comment by scaljeri** [comment]
This is indeed not possible. I have to investigate, but I think the only way this is possible is via the "Mocking with code" option

<img width="886" alt="mocking-with-code" src="https://user-images.githubusercontent.com/1078741/178957524-7f87d83f-628d-4149-b43d-027594214875.png">

Then you can use FETCH like this 

<img width="900" alt="mocking-coe-fetch" src="https://user-images.githubusercontent.com/1078741/178959575-a028ba5f-2a5d-4bc7-8d0b-909eca915ba9.png">

At the moment using fetch will probably give you CORS errors because fetch is executed in a sandbox inside the chrome extension. The only way right now to fix this is to start you browser with web-security disabled. On mac the command looks like this

    $> open -n -a /Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --args --user-data-dir="/tmp/chrome_dev_test" --disable-web-security

But that might not be all,  because of this sandbox you might not have the required cookies. Cookie support is something I'm working on right now.  

So, long story short, after cookie support I will try to fix this issue!
